Lively/src/components/sidebar/DotIcon.tsx

```diff
@@ -0,0 +1,52 @@
+
+import * as React from 'react';
+import './DotIcon.css';
+import Tooltip from '../wrappers/Tooltip';
+
+interface DotIconProps {
+    status: string;
+}
+
+class DotIcon extends React.Component<DotIconProps, {}> {
+    statusTooltipData = (status: string) => {
+        if (status === 'active') {
+            return {
+                primary: 'Active',
+                orientation: 'se'
+            };
+        } else if (status === 'snooze') {
+            return {
+                primary: 'Away',
+                secondary: 'Notifications Snoozed',
+                orientation: 'se'
+            };
+        } else {
+            return {
+                primary: 'Away',
+                orientation: 'se'
+            };
+        }
+    }
+
+    iconFromStatus = (status: string) => {
+        if (status === 'active') {
+            return <i style={{'color': '#38978D'}} className="fa fa-circle" aria-hidden="true" />;
```

Comment:
in the modern day, nobody uses b and i tags anymore, they've been replaced with strong and em tags instead. i tags are more often used for fonticons nowadays